example/improve_xml.py

```diff
@@ -0,0 +1,129 @@
+"""
+Improving non-functional properties ::
+
+    python improve_xml.py ../sample/Triangle_fast_xml
```

Comment:
shouldn't it be `../sample/Triangle_fast_srcml` rather than `../sample/Triangle_fast_xml` to support the actual statement-level operations..?